bottlecap

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
```

Comment:
`cork` is nice since it's 4 letters :) - I like `cass` because it's also 4 letters, but I also like highlighting that this is the script vs an alias that someone has defined on the command line (since I like to define my alias as `alias cass=`. So `cass.sh` would be my suggestion here, but also takes us back to 7 characters so I'm a hypocrite :)

This was mostly just a thought excercise for me. We don't need to change anything unless you want to.